functions.php

```diff
@@ -23,7 +23,7 @@
  * Initialize the metabox class.
  */
 /* FUNCTION to check for posts having snippets */
-add_action('wp','check_snippet_existence','',7);
+add_action( 'wp','check_snippet_existence' );
 function check_snippet_existence(){	

```

Comment:
Add a prefix to function name to make this unique

Reply:
Okay